### PR TITLE
[protocol] remove solc warnings, fixes #915

### LINKED ISSUE #915: Developers SNBAT see solc warnings
### Expected Behavior

No solc warnings

### Current Behavior

Many solc warnings for `FixidityTest.sol`, `LockedGold.sol`

### PR DESCRIPTION
**Pull request by diminator**
### Description

Fix solc warnings by modifying functions to pure

### Other changes

update readme with right test command

### Related issues

- Fixes #915

### CODE REVIEW

File: packages/protocol/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ By default, `NAME` will be set as `RELEASE_NAME`, `NAMESPACE_NAME`, `TESTNET_NAM
 To test the smart contracts, run:
 
 ```bash
-yarn run test:local
+yarn run test
```
Comment (m-chrzan) [inline packages/protocol/README.md:75]: 👍

File: packages/protocol/contracts/common/test/FixidityTest.sol [hunk 2]
```diff
@@ -6,47 +6,91 @@ import "../FixidityLib.sol";
 contract FixidityTest {
   using FixidityLib for FixidityLib.Fraction;
 
-  function newFixed(uint256 a) external view returns (uint256) {
+  function newFixed(uint256 a)
```
Comment (m-chrzan) [inline packages/protocol/contracts/common/test/FixidityTest.sol]: Most, if not all, of these function headers should fit on one line each.

**Review by m-chrzan — COMMENTED** [review]
Just a small style nit

File: packages/protocol/contracts/common/test/FixidityTest.sol (hunk 2 above)
Comment (asaj) [reply packages/protocol/contracts/common/test/FixidityTest.sol]: +1, our line limit is 100 chars

**Review by asaj — APPROVED** [review]
LGTM pending outstanding comments

**Comment by diminator** [comment]
@asaj what's the merge strategy for PRs?

**Comment by diminator** [comment]
Hi @asaj - let me know if this one is good to merge

**Comment by asaj** [comment]
Looks good to merge! Just blocked on fixing the protocol unit tests on master, which @yorhodes is looking into

**Comment by yorhodes** [comment]
> Looks good to merge! Just blocked on fixing the protocol unit tests on master, which @yorhodes is looking into

Fixed pending merge of https://github.com/celo-org/celo-monorepo/pull/1084

**Review by m-chrzan — APPROVED** [review]

**Pull request merged**